src/components/uiKit/SetsManagementDropdown/UserSetForm/index.tsx

```diff
@@ -1,21 +1,37 @@
 import intl from 'react-intl-universal';
-import { UserOutlined } from '@ant-design/icons';
+import { ExperimentOutlined, FileTextOutlined, UserOutlined } from '@ant-design/icons';
 import { Col, Form, Row, Select } from 'antd';
 import { FormInstance } from 'antd/lib/form';
 import { Store } from 'antd/lib/form/interface';
 
+import LineStyleIcon from 'components/Icons/LineStyleIcon';
 import { IUserSetOutput, SetType } from 'services/api/savedSet/models';
 
 import styles from './index.module.scss';
 
-type OwnProps = {
+const UserSetsFormIcon = ({ type }: { type: SetType }) => {
+  switch (type) {
+    case SetType.PARTICIPANT:
+      return <UserOutlined />;
+    case SetType.FILE:
+      return <FileTextOutlined />;
+    case SetType.BIOSPECIMEN:
+      return <ExperimentOutlined />;
+    case SetType.VARIANT:
+      return <LineStyleIcon height={16} className={styles.iconSvg} />;
+    default:
```

Comment:
un details pas obligé de corriger mais tu aurais pu écrire
```suggestion
   case SetType.PARTICIPANT:
   default:
      return <UserOutlined />;
   
```